### DISCUSSION TITLE
Thumbnail Fixer

### DISCUSSION THREAD

**Discussion by StefanOltmann** [lopcode/vips-ffm#130] [Show and tell]
Hi,

Thank you once again for the modifications you made for me. 😊

I'm excited to share that I now use vips-ffm in a new tool I've just released:
The Thumbnail Fixer.

This handy utility repairs and enhances embedded thumbnails in JPG, PNG, and WebP files, making them load faster in many applications.

You can check it out here:
https://github.com/stefanoltmann/thumbnail-fixer

In the past, I created a similar project privately, but its UI wasn't as polished, and it used SKIA, which was slower and more resource-intensive.

With the new UI and VIPS, I'm proud to share this with the world. 😊

Kind regards and happy holidays,

Stefan

**Comment by lopcode** [comment]
That looks great! Thanks so much for sharing
